cvat-ui/src/styles.scss

```diff
@@ -18,6 +18,10 @@ hr {
     margin: 10% 25%;
 }
 
+.cvat-text-color {
```

Comment:
the class name sounds too generic: does it really apply to all text in CVAT?
is there a specific meaning for this class?

Reply:
I've removed that class in my previous PR - https://github.com/daedaleanai/cvat/pull/11/files?#diff-3f857da70921ad42793f9f8e4eddf5e9L21. I've reverted some minor changes and have to add the class back. I want to leave it as it was before to not to diverge a lot from the upstream.